### ISSUE TITLE
Banned from Freenode

### ISSUE THREAD

**Issue by johnbentcope** [hashbang/shell-server#39]
Not sure where this goes, but I'd like to be transparent in my communication with their Ops.

===
Myself:

Could you elaborate on why our server was banned from connecting to Freenode? We have many many members, and that ban has suddenly affected everyone connected. Perhaps you could enlighten us to the nick of the offender? We'd appreciate working with Freenode  to ensure proper use of all systems, and to foster a more productive relationship.

Thank you for providing an awesome IRC network, and for reading this far. I hope you have a great day/night.

As always

(dsh80, found in #familab)

===
Freenode:

Good Morning,

I'm afraid one of the users of your service was observed exploiting other users
of the network. What kind of service is this server, if a ZNC or similar
bouncer, would it be possible for you to run an identd? That would allow easier
identification & removal of the specific user.

Regards,
Pricey

===
Myself:

That's very unfortunate. We host open shells for anybody curious about Linux to get their feet wet and get a taste for command line work. We also start people off with a weechat to get them used to IRC. I'm not sure what the current user count is, but it's nearing 100 people. We're not opposed to banning people from our network if they're causing trouble for others, we have a very solid be policy on respecting one another. If you could give us any info about the nicks being found in violation of your terms, we'd love to take care of the problem upstream, especially if they're exploiting others.

Thank you for your understanding  and communications on this matter.

**Comment by lrvick** [comment]
The solution here is to install a compatible identd server on all our shell boxes. A separate issue for that should be placed on the shell-etc repo.

Once that is in place we can respond letting them know we have identd in place which should make them feel much safer in re-allowing us as we can much more easily pinpoint bad actors.

**Comment by johnbentcope** [comment]
Oh awesome. Is there anything the user class people can do to help with that?

**Comment by lrvick** [comment]
@Alissaaquared and I have a solution I am deploying tonight.

On Thu, Jul 2, 2015 at 11:41 AM, John Cope <notifications@github.com> wrote:

> Oh awesome. Is there anything the user class people can do to help with
> that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashbang/shell-server/issues/39#issuecomment-118121389>
> .
>



-- 
Lance R. Vick
__________________________________________________
Cell      -  407.283.7596
Gtalk     -  lance@lrvick.net
Website   -  http://lrvick.net
PGP Key   -  http://lrvick.net/0x36C8AAA9.asc
keyserver -  subkeys.pgp.net
__________________________________________________

**Comment by AlissaSquared** [comment]
oidentd has been installed and configurations have been put in place so that all IRC connections to a server supporting ident checks will now be authenticated with a non-spoofable ident confirmation.
